meetings/2022-12-01.md

```diff
@@ -90,17 +102,132 @@ URL: https://github.com/solid/vocab/pull/62
+* eP: Why not notifications just coming at midnight?
+* RG: There is a start time from which you are ready to receive notifications and then i want the start time when to send notifications. connection to open before hand.
+* SC: I think this is a little over-engineered.
+* RG: server will give me a websocket url. i should be able to access at some point eg, 11 but don't want notifications until midnight.
+* eP: the scheduling should be optional.
+* SC: I don't think RG was focusing on required or optional
+* RG: I think this would be optional and we can defer it to next version 
+* SC: I think we should just change expiry to end time, if server advertises end time, we should advertise start in the same way.
+* RG: I would prefer that servers tells me when it will be sending notifications.
+* eP: IMO we should add inline issue linking to #111 so we communicate it as a feature under an active discussion
+* SC:
+  ```
+  #subscription request:
+  type: one
+  topic: at least one
+  target: zero or one (for target based subscriptions)
+  feature: zero or one per feature (#notification-features)
+    endTime (renamed from expiration -- proposed change)
+    startTime (proposed feature)
+
+  #subscription response:
+  type: one
+  topic: at least one
+  source: zero or one (for source based subscriptions)
+  sender: zero or one (for target based subscriptions)
+  feature: zero or one per feature (#notification-features)
+    startTime (zero or one)
+    endTime (zero or one: renamed from expiration -- proposed to require in the response)
+  ```
+* RG: would start time be a feature? I think all would be features in the request.
+* SC: If we just are leaving it as a future, what if we just push start time as a feature. I don't think any server is obligated to support specific features.
+* eP: Are you suggesting to have two different features, expiration and start time?
+* SC: We should rename expiration to end time.
+* eP: +1 to adjust it this way
+* RG: I think end time should be required in the response.
+* eP: subscription server should include endTime if it uses one. if feature is offered. it should be communicated if they want to leave it open (omit) but if determined they include endTime.
+* RG: Will client assume infinite, if missing?
+* SC: should be xsd valid if there is a value.
+* eP: for subscription type, we'd want to be able to unsubscribe. next release should.
+* SC: LDN uses inbox, in LDN the term target is used for the thing that advertises it inbox. I don't think notification protocol should add inbox. 
+* eP: We have different namespaces we do have `ldp:inbox` and we discuss defining `notify:target`. Do you consider also defining `notify:target`?
+* SC: https://solid.github.io/notifications/linkeddatanotifications-subscription-2021
+* eP: What benefit do you use in defining custom property for LDN subscription type? it could specify that `notify:target` points to an LDN inbox.
```

Comment:
```suggestion
* eP: What benefit do you see in defining custom property for LDN subscription type? It could specify that `notify:target` points to an LDN inbox.
```